sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/DocumentType.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/** Defines values for {@link DocumentType}. */
+@Immutable
+public final class DocumentType extends ExpandableStringEnum<DocumentType> {
```

Comment:
I was wondering about this one myself: Would it make sense to name this one "HealthcareDocumentType"?

Reply:
Yes. "DocumentType" is a name. @kristapratico Do you want to align with renaming "DocumentType" to "HealthcareDocumentType" since this renaming is been brought up on the table now?

Reply:
sure, seems fair

Reply:
This transform worked for me.

```
directive:
  - from: swagger-document
    where: $["definitions"]
    transform: >
      $["HealthcareTaskParameters"]["properties"]["documentType"]["x-ms-enum"]["name"] = "healthcareDocumentType";
```

Reply:
That sounds like something all languages can align on including JS. May I suggest that we incorporate that change into the swagger or at least check if the service team have a reason to omit the "Healthcare" part?

Reply:
Created a PR: https://github.com/Azure/azure-rest-api-specs/pull/21114